src/components/DefaultButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const DefaultButton = ({ content, additionalStyle, onClick, disabled }) => {
+    const buttonClassName =
+        `max-w-full w-40 mx-auto items-center py-2 text-white transition-all duration-500 hover:ring-8 disabled:cursor-not-allowed disabled:ring-0 ` +
```

Comment:
There is no need to explicitly declare a variable, just write to the `className` attribute of the `button`.

Reply:
You can also use `import clsx from 'clsx'` to concatenate class.